LICENSE.md

```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
+The Author may require that a Distributor provides a direct link to the Mod.  
 The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author may revoke distribution rights from any Distributor without prior notice.  
 
 6. Right to contribute
 --------
 
-The User may "fork" the Codebase.  
+The User may fork the Mod or any Software associated.  
 The User may compile the Codebase.  
 The User may submit contributions to the Author for inclusion in the Mod.  
 The User grants the Author all rights to any contribution.  
-The User renounces all rights to their contribution except as specified under this license.  
-The User retains the right to re-use code they've written.  
+The User renounces all rights to their contribution except as specified otherwise by the Author.  
```

Comment:
No. If anything, the whole "except as specified" bit should be removed. The option for the author to provide an additional amending contributor license agreement is sufficient. Legally, it's better to make a blanket statement then define the specific exceptions, rather than defining each case specifically.